Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM rust:1.64.0
+FROM rust:1.60.0
```

Comment:
we should stay in sync with rust-version.sh by running the ./cargo below instead of normal cargo

Reply:
this will be overridden during the build process. mainly changing this to prevent downloading rust again

Reply:
its already using `./cargo`: https://github.com/jito-foundation/geyser-grpc-plugin/blob/91f2ce54c3d29ae8c73265b87b96145b30082230/Dockerfile#L45

Reply:
yeah i see, mainly just point on the comment keeping it up to sync when its actually the ./cargo keeping it in sync

Reply:
updated comment